src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEvent.cs

```diff
@@ -21,7 +21,7 @@ public DiagnosticEvent(string hostId, DateTime timestamp)
 
         public int HitCount { get; set; }
 
-        public DateTime LastTimeStamp { get; set; }
+        public DateTimeOffset TimeStamp { get; set; }
```

Comment:
Should it be `Timestamp` ?

Reply:
Another place where it is called `Timestamp` instead of `TimeStamp` https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests.Shared/FunctionExecutionEvent.cs#L27

Reply:
actually this field needs to be removed as it is part of the TableEntity already. I think i missed adding one more commit

Reply:
the TimeStamp field needs to be removed as table entity already has it.